src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
```

Comment:
>Unreleased [](http://example.com/codeflow?start=3&length=10)

One thing need to be done is to add the build script for the test files I added, 
talked to Mary we suggested a location in this PR:
[https://github.com/microsoft/binskim/pull/530](https://github.com/microsoft/binskim/pull/530)

Once we settle the location I will add in this PR at the same location and format.

Reply:
done, added at the same place with same format.